presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -10,16 +10,102 @@ it does not cause running queries to fail; instead new queries become queued.
 A resource group may have sub-groups or may accept queries, but may not do both.
 
 The resource groups and associated selection rules are configured by a manager, which is pluggable.
-Add an ``etc/resource-groups.properties`` file with the following contents to enable
-the built-in manager that reads a JSON config file:
+
+There are two built-in managers: file-based and database-based. Add an ``etc/resource-groups.properties``
+file with the appropriate configurations to enable one of the two managers.
+
+File Resource Group Manager
+---------------------------
+
+This manager reads the resource group configuration from a JSON config file.
 
 .. code-block:: none
 
     resource-groups.configuration-manager=file
     resource-groups.config-file=etc/resource-groups.json
 
-Change the value of ``resource-groups.config-file`` to point to a JSON config file,
-which can be an absolute path, or a path relative to the Presto data directory.
+* ``resource-groups.config-file``: a JSON file storing the configurations, which can be an absolute
```

Comment:
The path to the JSON file can be an absolute path, or a path relative to the Presto data directory.